.gitignore

```diff
@@ -32,5 +28,16 @@ DerivedData
 # ignore snapshot failure files
 **/FailureDiffs/*
 
-KanvasExample/Pods
 
+/artifacts
+
+# Fastlane
+fastlane/README.md
+fastlane/test_output
+fastlane/report.xml
+
+# CocoaPods
+Example/Pods
```

Comment:
I can see the the `Pods/` folder is ignored here. However, this folder must have been tracked before this line was added because it appears in the diff.


![Untitled3](https://user-images.githubusercontent.com/1218433/177483273-2b2f1b87-1421-4232-a8c2-1750c829ff28.jpg)

I'm going to take the liberty to remove it then merge this PR. I'm confident in doing this because, in case I misunderstood something and we actually want to check the Pods into the repo, we can always add a new local copy and push it to the remote.